pydruid/utils/filters.py

```diff
@@ -79,11 +79,11 @@ def __init__(self, extraction_function=None, ordering="lexicographic", **args):
             self.filter["filter"].update(
                 {
                     "dimension": args["dimension"],
-                    "lower": args["lower"],
-                    "lowerStrict": args["lowerStrict"],
-                    "upper": args["upper"],
-                    "upperStrict": args["upperStrict"],
-                    "alphaNumeric": args["alphaNumeric"],
+                    "lower": args.get("lower"),
```

Comment:
Does `lower` and `upper` work with `None`? Per the [bound filter](https://druid.apache.org/docs/latest/querying/filters.html#bound-filter) documentation it seems these are both optional however there is no default specified. 

I think it would be prudent to check if a value of `None` is equivalent to not providing the key.

Reply:
I checked this, and it does not seem to be a problem, and they are equivalent.

Even in the Band class, Previously considered equal. [HERE](https://github.com/druid-io/pydruid/blob/54100504e9eb763f2f98da6c862179420baaf403/pydruid/utils/filters.py#L206)